src/org.xtuml.bp.x2m/src/org/xtuml/bp/x2m/Xtuml2Masl.java

```diff
@@ -128,8 +136,14 @@ public void xtuml2masl() throws IOException, RuntimeException {
         // pipe inputs and outputs together
         Path projectPath = new File(projectLocation).toPath();
         String projectName = projectPath.getName(projectPath.getNameCount() - 1).toString();
-        FileInputStream inputFile = new FileInputStream(
+        FileInputStream inputFile;
+        if (prebuild) {
+            inputFile = new FileInputStream(
                 projectLocation + File.separator + CODE_GEN_FOLDER + File.separator + projectName + ".sql");
```

Comment:
When prebuild output is used, does the model data get loaded twice?

Reply:
another thought is to pass the prebuilder output file.  ftw works on a single file as well as a directory